postgres/migrate-db/src/migrations/001.do.create_table_ingestion_events.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE ingestion_events (
+	ingest_id TEXT NOT NULL,
+	blob_id TEXT NOT NULL,
+	"type" TEXT NOT NULL,
+	status TEXT NOT NULL,
+	details JSONB,
+	event_time TIMESTAMP NOT NULL
+);
+CREATE TABLE blob_metadata (
+	ingest_id TEXT NOT NULL,
+	blob_id TEXT NOT NULL,
+	path TEXT NOT NULL, -- includes file name
+	file_size BIGINT NOT NULL,
+	insert_time TIMESTAMP NOT NULL
+);
+CREATE INDEX ON ingestion_events (ingest_id, blob_id);
+CREATE INDEX ON blob_metadata (ingest_id, blob_id);
+CREATE INDEX ON ingestion_events ("type");
+CREATE INDEX ON ingestion_events ((details->>'extractorName'));
```

Comment:
this last one isn't needed because the list of extractors will be so short once we've filtered to type=MimetypeDetected